frontend/test_setup/jestSetup.js

```diff
@@ -47,6 +47,9 @@ window.HTMLDivElement.prototype.getBoundingClientRect = function() {
 // process.on('unhandledRejection', (reason) => {
 //   console.log('REJECTION', reason)
 // })
+// process.on('uncaughtException', function(err) {
+//   console.log('EXCEPTION', err);
+// });
 
```

Comment:
Is there a reason why you commented this ? This might ring a bell in case of valid uncaughtException situation within the components unit/functional testing.

Reply:
If there's an uncaught exception in the app, it will be thrown in the console. But in case of tests - not always. Also right now there are some minor problems (promises not finished before the test is done) that are polluting the log, but I don't have time to look into.

Reply:
Opened https://github.com/metasfresh/mf15/issues/250 for that. Thanks